### PR TITLE
sidekick-browser: add version 104.29.3.25043-e335c0c

### LINKED ISSUE #9294: [Request] Sidekick
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** Sidekick

**Description:** Sidekick is designed for the ultimate online work experience and brings together every web tool you use.

**Homepage:** https://www.meetsidekick.com/

**Download link(s):** https://www.meetsidekick.com/download/

**Some indication of popularity/repute:** [GitHub stars/software reviews etc.]

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #9294

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by KrisSchneider** [comment]
@issaclin32 or @tech189. I am having a hard time figuring out the regex for the checkver in this manifest. When either of you have the time, could you please help me. 

This is the regex I used for this [website](https://www.softpedia.com/get/Internet/Browsers/Sidekick-PushPlayLabs.shtml). 
`([\d]+(\.[\d]+)+)([+-]?(?=\.\d|\d)(?:\d+)?(?:\.?\d*))(?:[eE]([+-]?\d+))?a\d`

**Comment by KrisSchneider** [comment]
The regex even works in VSCodium. So why does it not work in scoop?
image_5

**Comment by tech189** [comment]
It could be because Scoop uses the .NET version of Regex. But it would be better to use the website directly rather than the Softpedia mirror.

You'll need to write a checkver script that just prints the ultimate redirect from this page: https://api.meetsidekick.com/downloads/df/win. Then you can use the regex you've already written in a regex section in checkver to parse it. Then autoupdate can form the download from https://fast-cdn.meetsidekick.com/builds/sidekick-win-release-x64-$version-df.exe

Inkscape's manifest uses this code which you might find useful:
https://github.com/ScoopInstaller/Extras/blob/0520617216b7178916a91ac81ea81f1234856c3c/bucket/inkscape.json#L30

**Comment by KrisSchneider** [comment]
Thank you, @tech189, I'll try that method.

**Comment by KrisSchneider** [comment]
Still not passing checks...

File: bucket/sidekick-browser.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "104.29.3.25043-e335c0c",
+    "homepage": "https://www.meetsidekick.com/",
+    "description": "A work browser that is focused on productivity, speed, and security.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.meetsidekick.com/terms-of-use/"
+    },
+    "notes": "Settings are stored in: ~\\AppData\\Local\\Sidekick",
```
Comment (tech189) [inline bucket/sidekick-browser.json:9]: Is this necessary to show to the user on installation? The notes section is more for listing config files that need to be edited before the program is usable or for more immediate messages

Comment (Zliced13) [reply bucket/sidekick-browser.json:9]: Should I delete this section?

Comment (Zliced13) [reply bucket/sidekick-browser.json:9]: I'll delete it, there is no use for it.

Comment (Zliced13) [reply bucket/sidekick-browser.json]: Done.

Comment (tech189) [reply bucket/sidekick-browser.json]: It's good to note, though, if maintainers in the future are wondering whether the app needs to have any config persisted. So do bring it up in future PRs (put it in your PR description)!

Comment (Zliced13) [reply bucket/sidekick-browser.json]: oh ok good to know.

**Review by tech189 — APPROVED** [review]
Tested installation, checkver, all good.

**Comment by Zliced13** [comment]
Wait let me manually update it to the latest verrsion.

**Comment by Zliced13** [comment]
Oh, I guess the bot will update it automatically. In my bucket, the github-actions bot auto updated the sidekick-browser to version 104.30.1.25327-885b812.

**Comment by Zliced13** [comment]
That's why I wanted you to wait, so I could update the version in this PR to 104.30.1.25327-885b812.

**Comment by tech189** [comment]
I ran autoupdate locally and it didn't pick up any update, which is odd. Let's see whether Excavator will pick it up.

**Comment by Zliced13** [comment]
Yes, please run the excavator manually, it should update sidekick-browser to this version, 104.30.1.25327-885b812.

**Comment by tech189** [comment]
No need, I manually updated it here https://github.com/ScoopInstaller/Extras/commit/d07c47d4a307c317ff322aac08fb84c708959e58

**Comment by Zliced13** [comment]
Thank you

**Pull request merged**